files/build_templates/per_namespace/gbsyncd.service.j2

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description={{docker_container_name}} service
+Requires=database{% if multi_instance == 'true' %}@%i{% endif %}.service updategraph.service
+ConditionPathExists=!/usr/share/sonic/hwsku/gearbox_config.json
```

Comment:
does this condition hold true for all asic ?

Reply:
Yes. 
one can confirm the same from UT logs attached (which shows that required 3 gbsyncd@<asic#> got spawned for LC with 3 ASICs.